www/.htaccess

```diff
@@ -129,3 +129,6 @@ RewriteCond %{HTTPS} !=on
 # Redirect to https version
 RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
 
+# Redirect all docs translations to their English version
+RewriteCond $1 !=en
+RewriteRule ^.*docs/(\w\w(?:-\w\w)?)/(.*)$ {{site.baseurl}}/docs/en/$2 [R=301,L]
```

Comment:
1. Please consider using a 302 (temporary redirect) first, and then changing it to a 301 (permanent redirect) later. If you get the 301 wrong, it will be almost impossible to fix.
2. We have a mechanism for redirecting one language to another: https://github.com/apache/cordova-docs/blob/082c8d213615a012422e4f435bf6d4bcb846881e/www/.htaccess#L66. Have you considered just using that?